src/fhir.js

```diff
@@ -50,30 +50,31 @@ class FHIRWrapper {
 }
 
 class PatientSource {
-  constructor(filePathOrXML) {
+  constructor(filePathOrXML, shouldCheckProfile = false) {
```

Comment:
I've never wanted to convert this repo to TypeScript more than switching this to an object with well-defined types and just JavaScripting it up everywhere!

But nice, I agree. I think `requireProfileTagging` is good, as that language makes sense because the Patient Source will not find anything unless the tag is there when this option is enabled